04.bowling/rubybowling.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << s
+end
+
+point = 0
+frames.each_with_index do |item, i| # item=frame配列の値 i=要素の番号
+  if item[0] == 10 && i < 9 # strike
+    point = point + 10 + frames[i + 1][0] + frames[i + 1][1]
```

Comment:
```suggestion
    point = point + 10 + frames[i + 1].sum
```

[nits] `Array#sum` でもよさそうです